pom.xml

```diff
@@ -47,4 +47,35 @@
         <module>standalone</module>
         <module>tests</module>
     </modules>
+
+    <profiles>
+        <profile>
+            <id>release</id>
+            <properties>
+                <skipGpg>false</skipGpg>
+            </properties>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-gpg-plugin</artifactId>
+                        <version>${gpg.plugin.version}</version>
+                        <executions>
+                            <execution>
+                                <id>sign-artifacts</id>
+                                <phase>verify</phase>
+                                <goals>
+                                    <goal>sign</goal>
+                                </goals>
+                            </execution>
```

Comment:
We need to add the gpg parameters, right?

Reply:
I think Ale is talking about the parameters that you included to make it work with Ubuntu 18

Reply:
Of course. Thank you so much.
Done.